app/models/user.rb

```diff
@@ -8,6 +8,17 @@ class User < ActiveRecord::Base
   validates_presence_of :username # for display name
   validates_uniqueness_of :email
 
+  before_create :create_default_subscription
+
+  scope :subscriber_daily_digest, -> {
+    where(digest_subscriber: true, digest_frequency: "daily") }
+
+  scope :subscriber_weekly_digest, -> {
+    where( digest_subscriber: true, digest_frequency: "weekly") }
```

Comment:
Expression at 17, 65 should be on its own line.<br>Space inside parentheses detected.